src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: "Avenir Book";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: 300;
+}
+
+body {
+  margin-top: 20px;
+}
+
+.input {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-bottom: 20px;
+  font-size: 16px;
+  line-height: 22px;
+  height: 66px;
+  width: calc(100% - 62px);
+  padding-left: 62px;
+}
+
+.input,
+.input:hover,
+.input:focus,
+.input::placeholder {
```

Comment:
it means you use the same styles both for static state and dynamic one. 
You need to separate styles for ::hover and ::focus
Apply for other cases

Reply:
pay attention to your loop icon. When I hover at <input> the loop shouldn't be disappeared